### PR TITLE
Upgrade Skills page to grid display

### PR DESCRIPTION
**Pull request by jazeee**
Separate out the overview styles from the skills styles.

Clean up some overview styles.

Update skills tab to data grid.

## Screenies

| Old | new |
| ---- | ---- |
| image | image |
| image | image |

### CODE REVIEW

File: bin/main.html [hunk 1]
```diff
@@ -649,21 +649,23 @@
 </div>
 
 <!-- skill section -->
-<div class="skills section">
-  <div class="flex-stack">
+<div class="skills section epic-skills-tab">
+  <div class="flex-stack full-width">
+    <div class="center bold">Disciplines & Skills</div>
```
Comment (jazeee) [inline bin/main.html:654]: I wonder if we need this heading any more.

File: ui/basicTab/basicTab.scss [hunk 2]
```diff
@@ -1,46 +1,52 @@
...
+    }
+
+    > :first-child {
+      width: 120px;
+      padding-right: 4px;
+      text-align: right;
+    }
+
+    select.skillexpertise {
+      width: 40px;
```
Comment (jazeee) [inline ui/basicTab/basicTab.scss]: This style was hidden in `epicSkills.scss`, now here.
No styles change in this file, just a reorg.

File: ui/components/gridTable/gridTable.scss [hunk 3]
```diff
@@ -84,10 +84,10 @@ $grid-column-gap: 2px;
 
   .epic-table-header-grid {
     .skillTP-grid {
-      color: darken($tp-color, 10%);
+      color: lighten($tp-color, 25%);
     }
     .skillCP-grid {
-      color: darken($cp-color, 10%);
+      color: darken($cp-color, 20%);
```
Comment (jazeee) [inline ui/components/gridTable/gridTable.scss:90]: The header CP and TP colors were a bit harder to see.
Now:
image

File: ui/epicSkills/epicSkills.scss [hunk 4]
```diff
@@ -1,145 +1,29 @@
...
-.skills-overview div[data-groupname] span {
-  display: inline-block;
-}
-/* overview section */
-
-.skills-overview .skillname {
-  width: 100px;
-}
-
-.skills-overview>div[data-groupname] .skillname {
```
Comment (jazeee) [inline ui/epicSkills/epicSkills.scss:71]: Overview styles now in overviewTab.scss

File: ui/basicTab/basicTab.scss [hunk 5]
```diff
@@ -1,46 +1,54 @@
...
+    width: $input-field-width;
+    margin-top: -1px;
+  }
+
+  .extraadvantageCPdescription {
+    display: inline-block;
+    text-align: center;
+    width: 26px;
+  }
 }
```
Comment (jazeee) [inline ui/basicTab/basicTab.scss:54]: No real style changes, just reorg.
image

File: ui/basicTab/basicTab.scss [hunk 6]
```diff
@@ -1,46 +1,54 @@
...
-  width: 40px;
-  margin-top: -1px;
-}
-
-.advantages .extraadvantageCPdescription {
-  display: inline-block;
-  text-align: center;
-  width: 26px;
+.advantages {
+  $input-field-width: 40px;
```
Comment (jazeee) [inline ui/basicTab/basicTab.scss:5]: Extracted out a constant.

**Review by johnlamping — APPROVED** [review]

**Pull request merged**